plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/config/Settings.java

```diff
@@ -7,9 +7,10 @@
     public static final String DEFAULT_PLUGIN_DIR_LOCATION;
     public static final String DEFAULT_WAR;
     public static final URL DEFAULT_UPDATE_CENTER;
-    public static final String DEFAULT_UPDATE_CENTER_LOCATION = "https://updates.jenkins.io";
+    public static final String DEFAULT_UPDATE_CENTER_FILENAME = "/update-center.json";
```

Comment:
We should default to the URL that gives valid JSON, users can override for the JSON-P urls
```suggestion
    public static final String DEFAULT_UPDATE_CENTER_FILENAME = "/update-center.actual.json";
```

Reply:
All are changed.